support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -34,6 +34,9 @@ import type { Title } from 'helpers/user/details';
 
 // ----- Types ----- //
 
+export type StripePaymentMethod = 'StripeCheckout' | 'StripeApplePay' | 'StripePaymentRequestButton' | 'StripeElements';
```

Comment:
We should check later whether there's a way to see if people are using Google Pay as part of the payment request button. But don't let that hold up this going out.

Reply:
good point, will create a card